src/common/storage/util.js

```diff
@@ -13,7 +13,17 @@ define([
 ], function (CONSTANTS, jsonPatcher, Q, REGEXP, generateKey) {
     'use strict';
 
-    function _getRootHash(project, parameters) {
+    /**
+     * @param {ProjectInterface} project
+     * @param {object} parameters - Specifies which project tree should be serialized:
```

Comment:
this description seems misleading (at least not in par with the name of the function)